Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/PointerData/InteractablePointerDataManager.cs

```diff
@@ -0,0 +1,257 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// Manages InteractablePointerData for Interactive
+    /// Converts HandTrackingInputEventData, input data and pointer data to InteractablePointerData
+    /// </summary>
+    public class InteractablePointerDataManager
+    {
+        protected Dictionary<uint, InteractablePointerData> pointerData = new Dictionary<uint, InteractablePointerData>();
```

Comment:
Explain what the map keys are. Is it pointer id to pointer?

Reply:
yes and removed.